pkger/models.go

```diff
@@ -747,6 +711,7 @@ func (s *SummaryCheck) UnmarshalJSON(b []byte) error {
 type SummaryDashboard struct {
 	ID          SafeID         `json:"id"`
 	OrgID       SafeID         `json:"orgID"`
+	PkgName     string         `json:"pkgName"`
```

Comment:
PkgName's are being added to identify a resource to a precise match within a package.